XRTK-Core/Packages/com.xrtk.core/Interfaces/IMixedRealityCameraRig.cs

```diff
@@ -26,9 +26,19 @@ public interface IMixedRealityCameraRig
         Camera PlayerCamera { get; }
 
         /// <summary>
-        /// The player's body transform. This <see cref="Transform"/> is synced to the player's head X &amp; Z values
+        /// Initial head position as reported by the Vendor API, for use when Unity does not manage the players head height
+        /// </summary>
+        Vector3 InitialHeadPosition { get; set; }
```

Comment:
Let's replace this with `HeadHeight` and make it a readonly property.

I'm not sure the initial head position is useful in the interface. It's likely just an implementation detail.